clouds/aws/utilities/rosa_cluster_driver.py

```diff
@@ -0,0 +1,68 @@
+import os
+from datetime import datetime, timedelta
+
+from rosa.cli import execute
+from simple_logger.logger import get_logger
+
+LOGGER = get_logger(name=__name__)
+
+
+def build_command_with_params(
+    cluster_name,
+    cluster_expiration,
+    replicas,
+    compute_machine_type,
+    openshift_version,
+    aws_region,
+    channel_group,
+):
+    return (
+        "create cluster "
+        f"--cluster-name={cluster_name} "
+        "--sts "
+        f"--region={aws_region} "
+        f"--channel-group={channel_group} "
+        f"--version={openshift_version} "
+        f"--expiration-time={cluster_expiration}Z "
+        f"--replicas={replicas} "
+        f"--compute-machine-type={compute_machine_type}"
+    )
+
+
+def main():
```

Comment:
Let's make it a proper cli tool, please use click to get the user params not only from env.